src/depends/common/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
 add_library (Common SHARED Common.cpp CommonData.cpp CommonIO.cpp FileSystem.cpp FixedHash.cpp RLP.cpp SHA3.cpp)
+include_directories(${Boost_INCLUDE_DIRS})
```

Comment:
:bulb: Given the number of places this is used, it seems to make sense to move it to the top level.